source/code/source/playfab/PlayFabEventBuffer.cpp

```diff
@@ -35,7 +35,7 @@ namespace PlayFab
 
         // First event is just a stub which is used only to maintain the initial consistency of m_head/m_tail.
         // It is immediately considered already consumed.
-        auto index = eventIndex->load(std::memory_order_relaxed);
+        auto index = static_cast<uint64_t>(eventIndex->load(std::memory_order_relaxed));
```

Comment:
> static_cast<uint64_t> [](start = 20, length = 22)

the static cast should be unnecessary. eventIndex is already a pointer to an atomic u64.

recommend removing the static cast and just writing the uint64_t type explicitly on the left-hand side